test/e2e/specs/editor/various/embedding.spec.js

```diff
@@ -122,7 +122,7 @@ test.describe( 'Embedding content', () => {
 		await expect(
 			currenEmbedBlock.getByRole( 'textbox', { name: 'Embed URL' } ),
 			'WordPress invalid content. Should render failed, edit state.'
-		).toHaveValue( 'https://wordpress.org/gutenberg/handbook/' );
+		).toHaveValue( 'https://wordpress.org/gutenberg/handbook' );
```

Comment:
Personally, I think that if the embed fails even if the URL does not have a trailing slash, we should restore it to the original URL.

However, that implementation is not done in #14705 and will require major modifications. If necessary, I think we can respond with follow-up PR.